src/main/java/nextstep/subway/line/domain/Sections.java

```diff
@@ -0,0 +1,129 @@
+package nextstep.subway.line.domain;
+
+import nextstep.subway.line.exception.SectionAddException;
+import nextstep.subway.line.exception.SectionSizeMinimunException;
+import nextstep.subway.station.domain.Station;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+@Embeddable
+public class Sections {
+    @OneToMany(mappedBy = "line", cascade = {CascadeType.PERSIST, CascadeType.MERGE}, orphanRemoval = true)
+    List<Section> sections = new ArrayList<>();
+
+    public List<Section> getSections() {
+        return sections;
+    }
+
+    public void add(Section section) {
+        sections.add(section);
+    }
+
+    public List<Station> getStations() {
+        if (sections.isEmpty()) {
+            return Arrays.asList();
```

Comment:
```suggestion
            return Collections.emptyList();
```
이렇게 하면 의도를 더 잘 드러낼 수 있지 않을까요?
asList()는 매개변수로 배열을 받는데, 입력하지 않은 경우 빈 배열을 생성하기 때문에 메모리 낭비가 됩니다.